src/MessageHandler/AttachEntryEmbedHandler.php

```diff
@@ -57,7 +57,9 @@ public function __invoke(EntryEmbedMessage $message): void
 
         $entry->type = $type;
         $entry->hasEmbed = $html || $isImage;
-        $entry->image = $cover;
+        if ($cover) {
```

Comment:
Above we already have a `!$cover &` check, this this not redundant?

Reply:
no it is not, because it is enough if one of the other two is set for it to overwrite the image even if there is not a cover

Reply:
ah you're correct, these are `&&` checks. My bad.